low-level/framework/utils/file_utils.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import os
+import shutil
+from cortx.utils.process import SimpleProcess
+from framework.utils.service_logging import logger
+
+
+class FileUtils:
+    """Handle file operations."""
+
+    @staticmethod
+    def delete_or_truncate_files(path, fformat=None, del_file=False, del_dir=False):
+        """Clean log files and delete files from dir."""
```

Comment:
Lets have multi docstring. Explain the parameters used in this method.

Reply:
Done.